pkgs/development/tools/build-managers/cmake/default.nix

```diff
@@ -58,6 +61,7 @@ stdenv.mkDerivation rec {
       "--no-system-jsoncpp"
     ]
     ++ optional (!stdenv.isCygwin) "--system-libs"
```

Comment:
Yes, I'm just wondering why we give cmake a bunch of unnecessary dependencies, instead of just adding those to expressions that need them.